packages/office-ui-fabric-react/src/components/Keytip/Keytip.styles.ts

```diff
@@ -1,5 +1,7 @@
 import { IKeytipStyleProps, IKeytipStyles } from './Keytip.types';
-import { ICalloutContentStyleProps, ICalloutContentStyles } from 'office-ui-fabric-react/lib/Callout';
+import { ICalloutContentStyleProps, ICalloutContentStyles } from '../../Callout';
+import { IStyleFunction, IPoint } from '../../Utilities';
+import { mergeStyleSets } from '../../Styling';
 
 export const getStyles = (props: IKeytipStyleProps): IKeytipStyles => {
```

Comment:
I'm not seeing any HC styling for keytips, when disabled the color should be grayText and the opacity should be 1